### ISSUE TITLE
Change the size and angle of the sticker use finger on screen

### ISSUE THREAD

**Issue by Vuinfang** [dbarner1/react-native-stickers#10]
Is there a way for the user to drag and drop to change the size and angle of the sticker?

**Comment by mrxain699** [comment]
> Is there a way for the user to drag and drop to change the size and angle of the sticker?
Did you find a solution? if yes kindly let me know I will be very thankful I am stuck on this problem for many days.

**Comment by Vuinfang** [comment]
Yep, I solved it with using react native PanResponder, also there is another library you can have a look---react native reanimated 2
